test/UI/conftest.py

```diff
@@ -13,35 +16,36 @@
 
 @pytest.fixture
 def driver(browser, request):
-    if browser == "chrome":
-        chrome_options = install_extension()
-        driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()), options=chrome_options)
-        wait_until_ad_blocker_installed(driver)
-        window_handles = driver.window_handles
-        driver.switch_to.window(window_handles[1])
-        driver.close()
-        driver.switch_to.window(window_handles[0])
-    elif browser == "firefox":
-        capabilities = {
-            "browserName": "firefox",
-            "browserVersion": "88.0",
-            "selenoid:options": {
-                "enableVNC": True,
-                "enableVideo": False
+    match browser:
+        case "chrome":
+            driver = init_chrome()
+        case "firefox":
+            capabilities = {
+                "browserName": "firefox",
+                "browserVersion": "88.0",
+                "selenoid:options": {
+                    "enableVNC": True,
+                    "enableVideo": False
+                }
             }
-        }
-        driver = webdriver.Remote(
-            command_executor="http://192.168.0.1",
-            desired_capabilities=capabilities)
+            driver = webdriver.Remote(
+                command_executor="http://192.168.0.1",
+                desired_capabilities=capabilities)
     driver.maximize_window()
-    driver.implicitly_wait(7)
+    driver.implicitly_wait(5)
```

Comment:
Suggestion: add implicit_wait constant to the `base_utils` file.
```suggestion
    driver.implicitly_wait(DEFAULT_IMPLICIT_WAIT)
```

Reply:
done